subsys/tracing/user/tracing_user.c

```diff
@@ -19,24 +19,30 @@ void __weak sys_trace_idle_user(void) {}
 
 void sys_trace_k_thread_switched_in(void)
 {
+/* FIXME: Limitation of the current x86 EFI cosnole implementation. */
+#if !defined(CONFIG_X86_EFI_CONSOLE) && !defined(CONFIG_UART_CONSOLE)
+
```

Comment:
I wonder if it might be more appropriate to use `IS_ENABLED()` here as well as in the function below.

Reply:
I think IS_ENABLED() is not target for this scenario, most use in the "if( IS_ENABLE(CONFIG_xxx))" code, thanks.